SteamReview/SteamReview/Main.cpp

```diff
@@ -0,0 +1,7 @@
+#include "Steam.h"
+
+int main()
+{
+	FSteam Steam = FSteam();
```

Comment:
Remember that the ` = FSteam()` is fine but unnecessary. Simply having `FSteam Steam;` will call the default constructor.